### ISSUE TITLE
Server Crash when breaking crafting multiblock blocks

### ISSUE THREAD

**Issue by Seefinn** [TechReborn/RebornStorage#71]
Minecraft version: 1.12.2
Forge: 14.23.1.255
Reborn Storage: 1.12.2-3.0.2.32
Reborn Core: 1.12.2-3.5.1.166

Issue:

When attempting to break part of the crafting multiblock, it attempts to place cable in the same block space after breaking. This cable does not exist, and when broken disappears, but does not drop an item. Attempting to place another block in that block space causes the block to not place, but the cable to reappear. If you are near where this happens, you cannot log back in to the server. Upon attempting to restart the server, it crashes on startup until the world is loaded from a backup. 

These multiblock blocks were placed before updating to the above versions of Reborn Storage and Reborn Core, the versions they were placed under are as follows:

Reborn Storage: 1.12.2-3.0.0.23
Reborn Core: 1.12.2-3.5.0.162

Crash log:

[crash-2017-12-08_21.53.09-server.txt](https://github.com/TechReborn/RebornStorage/files/1544325/crash-2017-12-08_21.53.09-server.txt)

**Comment by Seefinn** [comment]
I should add that these two blocks are the only two I've found, but I've not tested all blocks in the multiblock. I had built a 15x15 shell before realizing that air was not a valid part and was attempting to remove the multiblock so I could shrink it to a point that I can afford to fill it.

I attempted to remove all of these blocks via builder, and it did pick them all up successfully, but this error still occurred.

**Comment by uehlbran** [comment]
Can confirm this mod has corrupted two worlds after breaking a crafting frame and then replacing it corrupts the world. It puts a cable, but even if you break the cable your world is done.

Stay away from this mod. Remove the mod and your world will load fine.

@gigabit101, Come on man...

Reborn Core: 1.12.2-3.5.0.162
Reborn Storage: 1.12.2-3.0.0.23

https://pastebin.com/A3uCQTrT

**Comment by uehlbran** [comment]
Also tested this mod with the latest versions of reborn storage and reborn core.

RebornCore-1.12.2-3.5.1.166-universal.jar 
RebornStorage-1.12.2-3.0.2.32.jar

Couldn't reproduce problem-- yet. Hopefully it's fixed.

**Comment by modmuss50** [comment]
Update the mod. RebornStorage-1.12.2-3.0.2.32.jar is the latest version.

**Comment by Seefinn** [comment]
I have updated the mod, as stated in my original post. I am on the latest version and this is still occurring. It started on an older version, and I updated as I saw another issue for this on this tracker stating that this was resolved in the latest version. It is not, I can only fix the error when it occurs with version 3.0.2.32 installed by either rolling the server back, or removing the mod entirely.

**Comment by modmuss50** [comment]
Can you post the log that you are getting when using the latest version?

**Comment by Seefinn** [comment]
The log in the original post is from using the latest version.

**Comment by modmuss50** [comment]
Ok, so that is the same as https://github.com/TechReborn/RebornStorage/issues/70 Im not sure what is causing it. I am waiting on a reply from the RS developer to see if he has any ideas. Sorry about this.

**Comment by Seefinn** [comment]
Alright, as that is still open I will watch that for updates on this. Thank you.
